### DISCUSSION TITLE
Cassandra migration failed

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#301] [Installation] [answered]
Hi 

CSP:GCP
Branch:- `release-4.10.0_RC1`

Cassandra migration failed 1.90 with null unable to proceed ahead for `Org channel creation`

![Screenshot from 2022-12-12 15-39-57](https://user-images.githubusercontent.com/20708426/207019017-2853557f-ad5c-4e8e-ab31-537265728a19.png)

**Comment by tsprasath** [comment, marked as answer]
[Cassandra-migration.txt](https://github.com/Sunbird-Ed/Community/files/10207623/Cassandra-migration.txt)
Attached the migration jenkins jobs logs.

**Reply by pkranga** [reply]
It says "success" as true for the 1.90 version as well. so which last version did it run successfully

**Reply by pkranga** [reply]
what is the error in org channel creation? @tsprasath

**Reply by tsprasath** [reply]
Cassandra's migration was successful.Unable to Channel.
